code/obj/machinery/computer/announcement.dm

```diff
@@ -225,9 +225,12 @@
 		var/job = person.mind.assigned_role
 		if(!job || job == "MODE")
 			job = "Staff Assistant"
+		if(ishivebot(person) || isrobot(person))
```

Comment:
Could probably just be `issilicon`?

Reply:
Testing it right now.

Reply:
Yeah that worked, simplified it and synced changed.